src/Microsoft.VisualStudio.Editors/PublicAPI.Shipped.txt

```diff
@@ -261,14 +261,11 @@ Microsoft.VisualStudio.Editors.MyApplication.MyApplicationProperties.SingleInsta
 Microsoft.VisualStudio.Editors.MyApplication.MyApplicationProperties.SplashScreen() -> String
 Microsoft.VisualStudio.Editors.MyApplication.MyApplicationProperties.SplashScreen(value As String) -> Void
 Microsoft.VisualStudio.Editors.OptionPages.GeneralOptions
-Microsoft.VisualStudio.Editors.OptionPages.GeneralOptions.CanChangeUsePreviewSdk() -> Boolean
```

Comment:
I am not familiar with how this file gets used but this looks like we removed a public API. Is that a public API as in externals (to Microsoft) can access it? David - Do you know?

Reply:
Maybe Jonathon know @ 

---
In reply to: [272756491](https://github.com/dotnet/project-system/pull/4696#discussion_r272756491) [](ancestors = 272756491)

Reply:
I wouldn't worry about it - this was new in 15.9 anyway.